### DISCUSSION TITLE
What is the ETA for a Laravel 10 supported release?

### DISCUSSION THREAD

**Discussion by timmeijers** [jenssegers/laravel-mongodb#2583] [General]
Last i heard is that in February the dev-master branch was not yet stable enough to release as is, now almost 7 months later still no update on when it will be released.

I am probably not the only one waiting for this, so please can you give us an indication on when you think it would be stable enough.

The alternative is that we would use the dev-master branch, which seems to work fine for us but then we would have to fork it so it wont be affected by random changes. It is not ideal...

**Comment by GromNaN** [comment]
I understand it's been a long wait, the release is planned for the next few weeks. https://github.com/mongodb-php/laravel-mongodb/issues/2499#issuecomment-1687960205
